3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -41,7 +28,15 @@ enrich {
     secret-key: "iam"
   }
 
-  # Kafka configuration
+  # config for NSQ sink and NSQ source
+  NSQ {
+    good-source: "{{enrichNSQSourceTopicGoodName}}" #topic name for good event source
+    bad-sink: "{{enrichNSQSinkTopicBadName}}" #topic name for bad event sink
+    nsqd-host: "{{NSQDHost}}" #host name for NSQD
```

Comment:
There are multiple tools on the NSQ project and NSQ server named as nsqd which means that NSQ Daemon. Also, there are other tools like nsqlookupd which finds correct nsqd instance from the topic name. Therefore, I used nsqd-, instead of nsq-